### ISSUE TITLE
ApiExplorer: Display input example when input is a complex type

### ISSUE THREAD

**Issue by bsschwa** [AmbitEnergyLabs/Nano#34]
The ApiExplorer should display a JSON example input next to the method when the input is a complex type.  The metadata returned by the Nano GetNanoMetadata method provides all the information necessary to produce this client-side.

**Comment by randyburden** [comment]
Yep, while we are at it we should also display example output in JSON as well. There is plenty of room on the right hand side of each ApiExplorer form for it.

Also take a look at how Swagger handles this same concept. They actually include both the JSON schema ( which is the example JSON you are wanting ) along with what they call a "Model" which kind of looks like their own custom JSON schema metadata format: http://petstore.swagger.io/#!/pet/addPet
